lib/core.js

```diff
@@ -2410,7 +2410,9 @@ declare var module: {
     loaded: boolean,
     parent: any,
     children: Array<any>,
-    builtinModules: Array<string>,
+    path: string,
+    paths: Array<string>,
+    isPreloading: boolean,
```

Comment:
It's not super clear why this is in `core.js` not `node.js`.  Most of these properties are specific to node's CommonJS implementation (have you ever read the [spec](http://wiki.commonjs.org/wiki/Modules/1.1)?  it's very cute), both before and after my changes, and I doubt flow targets any other implementations.

Let me know if you'd like me to move it.  That being said, I have no idea what the significance of things being in different files even is, if any.